rllab/envs/dm_control_env.py

```diff
@@ -0,0 +1,82 @@
+import pygame
+import numpy as np
+
+from dm_control import suite
+from dm_control.rl.environment import StepType
+
+from rllab.envs.base import Env, Step
+from rllab.envs.dm_control_view import DmControlViewer
+from rllab.core.serializable import Serializable
+from rllab.spaces.box import Box
+'''
+This environment will use dm_control toolkit(https://arxiv.org/pdf/1801.00690.pdf) 
+to train and simulate your models.
+'''
+
+
+def _flat_shape(observation):
+    return np.sum( int(np.prod(v.shape)) for k,v in observation.items())
+
+
+def _flat_observation(observation):
```

Comment:
https://github.com/ryanjulian/rllab/blob/integration/rllab/misc/tensor_utils.py

Reply:
Actually, if you should probably use this https://github.com/deepmind/dm_control/blob/master/dm_control/rl/control.py#L369